code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -40,7 +40,7 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.2.0
+  version: 0.9.2
```

Comment:
We follow OpenAPI Specification [https://spec.openapis.org/oas/v3.0.3#format](url). In addition other groups are using the same structure.

![image](https://github.com/camaraproject/EdgeCloud/assets/150795747/fd71a858-4de3-4722-aa82-52416de5c758)

Reply:
I was just suggesting changing the order of the fields, so that version appears before the long description, making it more prominent. I don't think the openapi spec requires fields to be listed in a specific order. In fact that's kind of a random order in the above image, I would have expected either alphabetical order, or perhaps required fields first. But not a big deal.